mainnet-canisters.json

```diff
@@ -79,6 +79,10 @@
     "rev": "e54d3fa34ded227c885d04e64505fa4b5d564743",
     "sha256": "3d808fa63a3d8ebd4510c0400aa078e99a31afaa0515f0b68778f929ce4b2a46"
   },
+  "sns_ledger_v2": {
```

Comment:
So does `sns_ledger_v2` point to ledger V3? :-)

Reply:
This points to V2 of the ledger wrt. the migration to stable structures as define in [the design doc](https://docs.google.com/document/d/1v34PJ0sgYKIKKF2r4o_qhVaeSHqeUwhmR7XnuE5UB0o/edit?tab=t.0#heading=h.mhhrcapko22j), which supports reading both "raw" stable memory, as well as stable memory written using the `MemoryManager`, in the `post_upgrade`. In the `pre_upgrade`, the serialized ledger state is written using the `MemoryManager`.